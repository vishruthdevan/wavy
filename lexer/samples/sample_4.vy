scaleVolume = function(volume, scale) {
    return volume * scale
}

aud1 = load("input1.wav")

louder_aud1 = scaleVolume(aud1, 2.0)
fainter_aud1 = scaleVolume(aud1, 0.5)

export(louder_aud1, 'louder_aud1.wav')
export(fainter_aud1, 'fainter_aud1.wav')