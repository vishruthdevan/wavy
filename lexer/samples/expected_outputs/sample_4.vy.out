<IDENTIFIER, "scaleVolume">
<ASSIGN, "=">
<FUNCTION, "function">
<LPR, "(">
<IDENTIFIER, "volume">
<COMMA, ",">
<IDENTIFIER, "scale">
<RPR, ")">
<LBRACE, "{">
<RETURN, "return">
<IDENTIFIER, "volume">
<ASTERISK, "*">
<IDENTIFIER, "scale">
<RBRACE, "}">
<IDENTIFIER, "aud1">
<ASSIGN, "=">
<IDENTIFIER, "load">
<LPR, "(">
<STRING, "input1.wav">
<RPR, ")">
<IDENTIFIER, "louder_aud1">
<ASSIGN, "=">
<IDENTIFIER, "scaleVolume">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<FLOAT, "2.0">
<RPR, ")">
<IDENTIFIER, "fainter_aud1">
<ASSIGN, "=">
<IDENTIFIER, "scaleVolume">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<FLOAT, "0.5">
<RPR, ")">
<IDENTIFIER, "export">
<LPR, "(">
<IDENTIFIER, "louder_aud1">
<COMMA, ",">
<STRING, "louder_aud1.wav">
<RPR, ")">
<IDENTIFIER, "export">
<LPR, "(">
<IDENTIFIER, "fainter_aud1">
<COMMA, ",">
<STRING, "fainter_aud1.wav">
<RPR, ")">
<EOF, "">
